testing/testlib/models/test_decision_tree_spark_classifier.py

```diff
@@ -21,10 +21,10 @@ def tree_model() -> (DecisionTreeClassificationModel):
 
 
 @pytest.fixture()
-def spark_dtree(tree_model, dataset_spark) -> ShadowSparkTree:
+def spark_dtree(tree_model, dataset_spark_tf) -> ShadowSparkTree:
```

Comment:
Should this really be `tf`? Seems like it should be just spark or is somehow spark using tf underneath?

Reply:
dataset_spark_tf it's a fixture for the dataset used in training for Spark and TF. I used the same dataset for testing for both spark and TF. Looking in the conftest.py I see we have different dataset fixtures for each library...don't remember exactly why.. maybe we needed a different format.